docs/pages/admin-guides/teleport-policy/crown-jewels.mdx

```diff
@@ -16,7 +16,8 @@ Crown Jewels, and how to see permission changes for these resources.
 
 For an improved experience, we recommend using Crown Jewels in conjunction with Teleport local users or integrating with
 [Okta](../../enroll-resources/application-access/okta/okta.mdx) or [Microsoft Entra ID](./integrations/entra-id.mdx).
-This setup helps mitigate the spam generated by highly privileged ephemeral users created by Teleport Auth Connectors.
+This setup helps minimize the number of access path change entries generated when highly privileged ephemeral users
+log in via Teleport Auth Connectors..
```

Comment:
```suggestion
log in via Teleport authentication connectors.
```
To use the convention we use elsewhere in the docs

Reply:
Are we being overly pedantic here?

The product literally calls them Auth Connectors:

<img width="356" alt="image" src="https://github.com/user-attachments/assets/9ed53028-fb0f-4c4d-a176-fd0df81e878c">